index.html

```diff
@@ -129,14 +129,11 @@ <h2>contact</h2>
       </form>
       <img src="assets/office.jpg" alt="Hello agency headquaters">
       <p>
-        Our office is located<br/>
-        in the heart of Warsaw
-      </p>
-      <p>Feel free to say hello!</p>
-      <p>
-        Warsaw St. 23/21<br/>
-        03-323 Warsaw,<br/>
-        POLAND
+        <span>Our office is located in the heart of Warsaw</span>
+        <span>Feel free to say hello!</span>
+        <span>Warsaw St. 23/21</span>
+        <span>03-323 Warsaw</span>
+        <span>POLAND</span>
```

Comment:
One more thing: it is not a good idea to write text with all caps in markup, you can use CSS `text-transform: uppercase;` to make it look uppercase.

Reply:
Should I use just use uppercase for the first letter in the markup or skip uppercase at all?
What about when I want to have e.g. Warsaw displayed? 
Should I write `warsaw` and then `text-transform: capitalize`?

Reply:
No, uppercase for the first letter is fine. It is just all uppercase that is shouting in screen readers. 😄 "Warsaw" is fine.